### DISCUSSION TITLE
Question on comment line break

### DISCUSSION THREAD

**Discussion by hiensarahly** [walinejs/waline#189] [Q&A] [answered]
Hey @lizheming I wanted to tell you again that I like Waline a lot. It feels very natural and convenient to use. I’ve successfully migrated here!

I’m seeing this line break at the end of a sentence: 

![3D9E9D19-D61E-4335-BDF0-399307E77A60](https://user-images.githubusercontent.com/21299841/113433652-d428eb00-9409-11eb-8739-aa99280f8bb7.jpeg)

Please help.

**Comment by lizheming** [comment, marked as answer]
Hi @hiensarahly, thank you for your loving! The question you asked is a default stylesheet problem. You can add following css into your website to cover default style, then comment will wrap by word.

```css
.v[data-class=v] .vcards .vcard .vcontent {
  word-break: normal;
}
```
